app/src/components/Modals/activity-modal/activity-form-modal.tsx

```diff
@@ -168,13 +171,18 @@ const AddActivityModal: FC<AddActivityModalProps> = ({
 
     // so the issue here is that we need to have one inventoryValue for
     const requestData = {
-      activityData: {
-        co2_amount: gasValues[1].factor,
-        ch4_amount: gasValues[0].factor,
-        n2o_amount: gasValues[2].factor,
-        ...values,
+      activityData: methodology?.id.includes("direct-measure")
+        ? {
+            co2_amount: gasValues[1].factor,
+            ch4_amount: gasValues[0].factor,
+            n2o_amount: gasValues[2].factor,
```

Comment:
Why are we saving emissions factors as gas measurements?
I think there is a confusion here of emissions factors vs. directly measured total gas amounts (both of these are present for CO2, CH4, N2O etc., but they serve very different purposes in the calculations.